### PR TITLE
[KOGITO-2885] - Promote pipelines: add a promote.properties file

### PR DESCRIPTION
**Pull request by msmagnanijr**
See: https://issues.redhat.com/browse/KOGITO-2885

Signed-off-by: Mauricio Magnani <msmagnanijr@gmail.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Comment by msmagnanijr** [comment]
@radtriste  will review it before I test it on Jenkins RHBA.

File: Jenkinsfile.promote [hunk 1]
```diff
@@ -384,7 +405,19 @@ String getParamOrDeployProperty(String paramKey, String deployPropertyKey){
 //////////////////////////////////////////////////////////////////////////////
 
 boolean isRelease() {
-    return params.RELEASE || (getDeployProperty('release') == 'true')
+    return params.RELEASE || (getDeployProperty('release') == 'true')GIT_AUTHOR
```
Comment (radtriste) [inline Jenkinsfile.promote:408]:
```suggestion
    return params.RELEASE || (getDeployProperty('release') == 'true')
```

File: Jenkinsfile.promote [hunk 2]
```diff
@@ -384,7 +405,19 @@ String getParamOrDeployProperty(String paramKey, String deployPropertyKey){
...
+}
+
+String getRepoName(){
+    return env.GITHUB_REPO
+}
```
Comment (radtriste) [inline Jenkinsfile.promote:413]: where do you use this one ?

File: Jenkinsfile.promote [hunk 3]
```diff
@@ -384,7 +405,19 @@ String getParamOrDeployProperty(String paramKey, String deployPropertyKey){
...
+
+String getBuildUrl(){
```
Comment (radtriste) [inline Jenkinsfile.promote:415]: I guess you can also update `readDeployProperties` with that accessor ;)

File: Jenkinsfile.promote [hunk 4]
```diff
@@ -384,7 +405,19 @@ String getParamOrDeployProperty(String paramKey, String deployPropertyKey){
...
+    return params.DEPLOY_BUILD_URL
+}
+
+String getBotBranch(){
```
Comment (radtriste) [inline Jenkinsfile.promote:419]: where is it used ?

Comment (msmagnanijr) [reply Jenkinsfile.promote:419]: thanks! forgot to remove it.

File: Jenkinsfile.promote (hunk 2 above)
Comment (msmagnanijr) [reply Jenkinsfile.promote:413]: the same, forgot to remove.

**Review by ricardozanini — APPROVED** [review]

**Comment by radtriste** [comment]
@msmagnanijr any update there ?

**Comment by spolti** [comment]
@msmagnanijr Hi, is there any other change here or is it ready to merge?
needs rebase.

**Comment by spolti** [comment]
@radtriste is it still needed?

**Comment by radtriste** [comment]
Closing that one as a refactoring is in progress and the promote.properties should be handled automaticallyu

**Pull request closed without merging**